### ISSUE TITLE
Accessibility Issue: Voice Control may not be able to see comments moderation status.

### ISSUE THREAD

**Issue by zachspears** [wordpress-mobile/WordPress-iOS#17542]
### Expected behavior
Voice Control should be able to communicate important information such as the comments moderation status. 

### Actual behavior
Voice Control appears to only be able to communicate the options but not which is selected. 

![IMG_0293](https://user-images.githubusercontent.com/10124024/143095363-e171a92f-110e-47c6-8039-1c6fcc012477.PNG)

### Steps to reproduce the behavior
1. Enable Voice Control with Item Names Overlay. 
2. Open the WordPress app and select a comment for Moderation.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
